CONTRIBUTING.md

```diff
@@ -0,0 +1,101 @@
+# Contributing to PaperMC Docs
+
+Thank you for considering contributing to the PaperMC Docs repository! We appreciate your help in making our documentation better. 
+To ensure a smooth and collaborative contribution process, please follow the guidelines below.
+
+## Table of Contents
+1. [Introduction](#introduction)
+2. [How to add a page](#adding-a-page)
+3. [Submitting Contributions](#submitting-contributions)
+4. [Style Guidelines](#style-guidelines)
+5. [Code of Conduct](#code-of-conduct)
+
+## Introduction
+
+The PaperMC Docs repository contains the documentation for the PaperMC project, which is used to provide essential 
+information, instructions, and guidelines to our users, developers, and contributors. Your contributions play a 
+significant role in improving and maintaining the quality of our documentation.
+
+We use Docosaurus to build and deploy the documentation website. The Docs Page is hosted on Vercel, which supports automatic
+CI/CD deployments from the main branch. The website is available at [https://docs.papermc.io](https://docs.papermc.io).
+
+Using Docosaurus means that there are guides for how to write the documentation. You can find the Docosaurus documentation at
+[https://docusaurus.io/docs/category/guides](https://docusaurus.io/docs/category/guides). The documentation is written 
```

Comment:
Saying `The documentation` after saying where to find the Docusaurus documentation makes it sound like *Docusaurus'* documentation is written in markdown, when I think you are talking about our documentation. Maybe just make the first occurrence of the word Docusaurus a hyperlink to the docs.